models/UserDao.js

```diff
@@ -0,0 +1,29 @@
+import prisma from '../prisma';
+
+const createUser = async (nickname, email, hashedPw) => {
+  return await prisma.user.create({
+    data: {
+      nickname: `${nickname}`,
+      email: `${email}`,
+      password: `${hashedPw}`,
+    },
+  });
+};
+
+const findUser = async (email) => {
+  return await prisma.user.findMany({
+    where: {
+      email: `${email}`,
+    },
+  });
+};
+
+const findUserId = async (id) => {
+  return await prisma.user.findMany({
+    where: {
+      id: `${id}`,
+    },
+  });
+};
+
```

Comment:
```suggestion
const findUserById = async (id) => {
  return await prisma.user.findMany({
    where: {
      id: `${id}`,
    },
  });
};

```

`findUserId`는 어감상 유저의 `id`를 찾아오는 함수로 해석될 오해의 소지가 있습니다.
함수명 정정해주세요 :)